gems/pending/util/miq-process.rb

```diff
@@ -127,24 +127,7 @@ def self.processInfo(pid = nil)
   end
 
   def self.command_line(pid)
-    case Sys::Platform::IMPL
-    when :mswin, :mingw
```

Comment:
sys-proctable supports :cmdline on Windows. That's one of the "universal" members between all platforms.